<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>期货公司:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>默认</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simnow电信一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simnow电信二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simnow移动</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Simnow7*24小时</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>招金期货测试</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>行情地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Md_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>交易地址:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Td_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>BrokerId:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Broker_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>账    号:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Account_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>密    码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Password_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>产品名称:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AppID_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>授权编码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Authorization_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="5,18">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>产品信息:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ProductionInfo_Edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3,3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveAccount">
        <property name="text">
         <string>保存信息</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SavePassword">
        <property name="text">
         <string>保存密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,2,2,2,1">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_login">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_simnow_registered">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
